src/styles/components/logo-soup.scss

```diff
@@ -0,0 +1,41 @@
+.logo-soup {
+    background-color: $human-blue;
+    width: 100%; 
+    height: auto;
+    position: relative;
+    padding: 2rem;
+    display: grid;
+    grid-template-columns: 1fr;
+    grid-template-rows: auto;
+    place-items: center; 
+
+}
+.soup__header{
+    max-width: 70vw;
+    text-align: center;
+    color: white;
+    font-size: 2.5rem;
+}
+.soup__content {
+    max-width: 925px;
+    display: grid; 
+    place-items: center;
+    grid-template-columns: repeat(8, 1fr);
```

Comment:
`8` is a magic number here as this layout will then break if the # of logos changes. Instead, let's use `auto-fit`. Also, `grid-gap` can handle setting padding between elements and `width: 100%` will ensure that this grows to fit the available space (up to the 925px max-width).

```scss
    width: 100%;
    max-width: 925px;
    display: grid; 
    place-items: center;
    grid-template-columns: repeat(auto-fit, minmax(50px, 1fr));
    grid-template-rows: 1fr;
    grid-gap: 1em;
```

Reply:
change grid columns as suggested and add grid-gap. removed paddings for both desktop and phone screen.